x-pack/test_serverless/functional/test_suites/observability/cases/attachment_framework.ts

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect } from 'expect';
+import { ConnectorTypes } from '@kbn/cases-plugin/common/types/domain';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { createCase } from './helper/api';
+
+export default ({ getPageObject, getService }: FtrProviderContext) => {
+  const dashboard = getPageObject('dashboard');
+  const lens = getPageObject('lens');
+  const svlCommonNavigation = getPageObject('svlCommonNavigation');
+  const svlObltNavigation = getService('svlObltNavigation');
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+  const cases = getService('cases');
+  const find = getService('find');
+  const supertest = getService('supertest');
+  const retry = getService('retry');
+
+  describe('persistable attachment', () => {
+    describe('lens visualization', () => {
+      before(async () => {
+        await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
+        );
+
+        await svlObltNavigation.navigateToLandingPage();
+
+        await svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'dashboards' });
+
+        await dashboard.clickNewDashboard();
```

Comment:
nit: this whole block could be moved to helper utils

Reply:
As discussed offline, I am keeping this block in this file as we don't have shared helper utils across different serverless test suites. We might create one in future if needed.